### ISSUE TITLE
A question about train on single GPU

### ISSUE THREAD

**Issue by xibi777** [NVlabs/SegFormer#82]
I train b0_cityscapes on 1 GPU, and I only change the samples_per_gpu into 8. It performs not well, and it seems that the model overfit after 60000 iterations, because the mIoU get lower when keeping training. Should I also change my learning rate? Thanks!!!

**Comment by StepherMaker** [comment]
Hello, I have the same problem about it, I train the mit_b1 ade20k on single GPU, but the performance is lower about 7 percentage, should I change the learning rate?
